evm/src/fuzz/mod.rs

```diff
@@ -54,14 +56,21 @@ where
         should_fail: bool,
         errors: Option<&Abi>,
     ) -> FuzzTestResult {
-        let strat = fuzz_calldata(func);
-
         // Stores the consumed gas and calldata of every successful fuzz call
         let cases: RefCell<Vec<FuzzCase>> = RefCell::new(Default::default());
 
         // Stores the result of the last call
         let call: RefCell<RawCallResult> = RefCell::new(Default::default());
 
+        // Stores fuzz state for use with [fuzz_calldata_from_state]
+        let state: EvmFuzzState = EvmFuzzState::default();
+
+        // TODO: We should have a `FuzzerOpts` struct where we can configure the fuzzer. When we
+        // have that, we should add a way to configure strategy weights
+        let strat = proptest::strategy::Union::new_weighted(vec![
+            (60, fuzz_calldata(func.clone())),
+            (40, fuzz_calldata_from_state(func.clone(), state.clone())),
+        ]);
```

Comment:
Decided to not make this configurable yet since we're probably going to change the signature of `FuzzedExecutor::new` when we introduce a `FuzzerOpts` struct